pkg/instrumentation/annotation.go

```diff
@@ -31,6 +31,21 @@ const (
 	annotationInjectApacheHttpdContainersName = "instrumentation.opentelemetry.io/apache-httpd-container-names"
 	annotationInjectNginx                     = "instrumentation.opentelemetry.io/inject-nginx"
 	annotationInjectNginxContainersName       = "instrumentation.opentelemetry.io/inject-nginx-container-names"
+	annotationJMXJVM                          = "cloudwatch.aws.amazon.com/inject-jmx-jvm"
+	annotationJMXTomcat                       = "cloudwatch.aws.amazon.com/inject-jmx-tomcat"
+	annotationJMXKafka                        = "cloudwatch.aws.amazon.com/inject-jmx-kafka"
+	annotationJMXKafkaConsumer                = "cloudwatch.aws.amazon.com/inject-jmx-kafka-consumer"
+	annotationJMXKafkaProducer                = "cloudwatch.aws.amazon.com/inject-jmx-kafka-producer"
+)
+
+var (
+	JMXAnnotationInjectionMap = map[string]string{
```

Comment:
nit: Do we need this map? Couldn't we trim the prefix to get the target system?

Reply:
I don't like doing string splits / postfix to get values. I think this is cleaner.